webrtc-stats.html

```diff
@@ -183,10 +183,11 @@
           Lifetime of stats objects
         </h3>
         <p>
-          The general rule is that stats objects, once created, exist for the lifetime of the
-          PeerConnection that contains them, even when the underlying object they report on is
-          stopped or deleted. This is important in order to report consistently on short-lived
-          objects and to be able to report totals over the lifetime of a PeerConnection.
+          The general rule is that stats objects are created at the same time as the underlying
```

Comment:
I'd formulate this as "are considered to be created". It's a perfectly valid implementation strategy to create "frozen" stats objects when the underlying objects are destroyed, and creating the "live" stats objects when queried - in fact, that's an implementation strategy I've been meaning to suggest to @henbos.

The observable behavior should be indistinguishable from the text you suggest.